astropy/utils/tests/test_metadata.py

```diff
@@ -11,7 +9,7 @@
 from astropy.io import fits
 
 
-class OrderedDictSubclass(OrderedDict):
+class OrderedDictSubclass(dict):
```

Comment:
This one we should keep the same - it specifically tests what happens if someone subclasses `OrderedDict` (which should still be allowed)

Reply:
Makes sense. Thanks!